x/datadeal/keeper/deal.go

```diff
@@ -305,16 +290,16 @@ func (k Keeper) VerifyDataCertificate(ctx sdk.Context, validatorAddr sdk.AccAddr
 		return false, sdkerrors.Wrapf(types.ErrInvalidSignature, "%s", cert.GetSignature())
```

Comment:
Above this we are using the `isValid` variable like this:
```go
        isValid := validatorPubKey.VerifySignature(unSignedMarshaled, cert.GetSignature())
	if !isValid {
		return false, sdkerrors.Wrapf(types.ErrInvalidSignature, "%s", cert.GetSignature())
	}
```

If you're not going to reuse the `isValid` variable, how about changing it to something like this:
```go
	if !validatorPubKey.VerifySignature(unSignedMarshaled, cert.GetSignature()){
		return false, sdkerrors.Wrapf(types.ErrInvalidSignature, "%s", cert.GetSignature())
	}
```

Reply:
Good point. Thanks!